js/src/toast.js

```diff
@@ -166,6 +182,126 @@ class Toast {
+        return top
+      }, this._config.positionMargin)
+
+      if (this._config.position === POSITION_TOP_RIGHT) {
+        this._element.classList.add(POSITION_TOP_RIGHT)
+        styles.right = `${this._config.positionMargin}px`
+      } else if (this._config.position === POSITION_TOP_LEFT) {
+        this._element.classList.add(POSITION_TOP_LEFT)
+        styles.left = `${this._config.positionMargin}px`
+      } else {
+        const leftPx = this._getMiddleToastPosition()
+
+        this._element.classList.add(POSITION_TOP_CENTER)
+        styles.left = `${leftPx}px`
+      }
+
+      styles.top = `${top}px`
+      Manipulator.applyCss(this._element, styles)
+      return
+    }
+
+    const bottom = toastList.reduce((bottom, toastEl) => {
+      const { height, marginTop } = window.getComputedStyle(toastEl)
+
+      bottom += (parseInt(height, 10) + parseInt(marginTop, 10))
+      return bottom
+    }, this._config.positionMargin)
+
+    if (this._config.position === POSITION_BOTTOM_RIGHT) {
+      this._element.classList.add(POSITION_BOTTOM_RIGHT)
+      styles.right = `${this._config.positionMargin}px`
+    } else if (this._config.position === POSITION_BOTTOM_LEFT) {
+      this._element.classList.add(POSITION_BOTTOM_LEFT)
+      styles.left = `${this._config.positionMargin}px`
+    } else {
+      const leftPx = this._getMiddleToastPosition()
+
+      this._element.classList.add(POSITION_BOTTOM_CENTER)
+      styles.left = `${leftPx}px`
+    }
+
+    styles.bottom = `${bottom}px`
+    Manipulator.applyCss(this._element, styles)
+  }
+
+  _repositionExistingToasts() {
+    if (!this._element.parentNode) {
+      return
+    }
+
+    const toastList = SelectorEngine.find(`.toast.${this._config.position}`, this._element.parentNode)
+
+    toastList.forEach((toastEl, index) => {
+      const toastInstance = Toast.getInstance(toastEl)
+
+      if (toastInstance.config.position.indexOf('top-') > -1) {
+        let top = toastInstance.config.positionMargin
+
+        if (index) {
+          const previousToast = toastList[index - 1]
+          const { height, marginBottom } = window.getComputedStyle(previousToast)
+
+          top += (parseInt(height, 10) + parseInt(marginBottom, 10))
+        }
+
+        toastEl.style.top = `${top}px`
+      }
+
+      if (toastInstance.config.position.indexOf('bottom-') > -1) {
+        let bottom = toastInstance.config.positionMargin
+
+        if (index) {
+          const previousToast = toastList[index - 1]
+          const { height, marginTop } = window.getComputedStyle(previousToast)
+
+          bottom += (parseInt(height, 10) + parseInt(marginTop, 10))
+        }
+
+        toastEl.style.bottom = `${bottom}px`
+      }
+    })
+  }
+
+  _clearPositioning() {
+    Manipulator.applyCss(this._element, {
+      position: '',
+      right: '',
+      left: '',
+      bottom: '',
+      top: ''
+    })
+    this._element.classList.remove(POSITION_TOP_RIGHT)
```

Comment:
I think we can pass multiple classes here?

Reply:
I wasn't sure about it, but yep I'll do it 👍